src/components/main/main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
fix formatting.

Reply:
I don`t understand what exactly is wrong with the formatting in this line. I will be glad if you explain more precisely.